ETC.tsv

```diff
@@ -97,109 +97,109 @@ ETC_20150317_000096	$Although their bright color makes them pretty visible, it's
+ETC_20150317_000138	$Unlike other wraith monsters, these monsters were often evil souls even when they were living.
 ETC_20150317_000139	$Giant Spectre
 ETC_20150317_000140	$Spectre of Deceit
 ETC_20150317_000141	$Green Rikaus
 ETC_20150317_000142	$Spectron
 ETC_20150317_000143	$Spectre Master
 ETC_20150317_000144	$Thornball
-ETC_20150317_000145	This floating seed has transmutated, losing its outer fluff for a solid shell.
+ETC_20150317_000145	$This floating seed has transmutated, losing its outer fluff for a solid shell and thorns. However, its floating capability have stayed intact.
 ETC_20150317_000146	$Big Thornball
 ETC_20150317_000147	$Woodspirit
 ETC_20150317_000148	$Blue Woodspirit
 ETC_20150317_000149	$Green Woodspirit
 ETC_20150317_000150	$Woodman
-ETC_20150317_000151	It would be a real blessing if the Woodman couldn't move.
+ETC_20150317_000151	$The fact that the Woodman can't move is a real blessing.
 ETC_20150317_000152	$Blue Woodman
 ETC_20150317_000153	$Gray Woodman
 ETC_20150317_000154	$Red Woodman
 ETC_20150317_000155	$Dark Slime
-ETC_20150317_000156	Its slimy substance allows it to model itself into a prototype of the object it last encountered, thus the slime can exist in a variety of types.
+ETC_20150317_000156	$Although slimes have similar appearance, they vary in species due to difference between the materials they originated from.
 ETC_20150317_000157	$Rusrat
-ETC_20150317_000158	During the times of Zachariel the Great, these monsters were placed to stop intruders. Its name stems from grave robbers who encountered it.
+ETC_20150317_000158	$Starting from the Royal Mausoleum of Zachariel the Great, these guardians were placed in many ruins to stop intruders. Its name stems from grave robbers who encountered it.
 ETC_20150317_000159	$Madakia
-ETC_20150317_000160	Court sorcerers created the Madakia to fend off intruding grave robbers.
+ETC_20150317_000160	$Court wizards created the Madakia to fend off intruders in case they are not mere grave robbers.
 ETC_20150317_000161	$Wheelen
-ETC_20150317_000162	In order to watch over sealed ruins that lacked an overseer, monsters were created with a diversity of magic. The wheelen is one of these monsters.
+ETC_20150317_000162	$The Wheelen is one of the magical creations made to protect sealed ruins without an overseer.
 ETC_20150317_000163	$Wheelen 3
 ETC_20150317_000164	$Mauros
-ETC_20150317_000165	The stone Calvary of the Royal Mausoleum are magically assembled to withstand even the heavily armed knight.
+ETC_20150317_000165	$The stone calvary of the Royal Mausoleum are the magical creations to withstand even the heavily armed knight.
 ETC_20150317_000166	$Truffle
-ETC_20150317_000167	Though the reason is unknown, this monster has an extreme hatred towards humans. It loves to attack them for food and survival.
+ETC_20150317_000167	$Though the reason is unknown, this monster has an extreme hatred towards humans. It prefer attacking them to hunting its prey or pursuing its survival.
```

Comment:
"For reasons unknown, this monster holds an extreme hatred towards humans. It prefers to attack humans rather than hunt more suitable prey or look out for its own survival."